### DISCUSSION TITLE
Maven release versions

### DISCUSSION THREAD

**Discussion by marius-mada** [release-drafter/release-drafter#1381] [Ideas]
Hello, can you please add an option so that the draft title synchronize with maven release version?
So that we have the same version in title as we have in the maven.
Thank you!

**Comment by jetersen** [comment]
You can do this via GitHub actions, see https://github.com/jenkins-infra/github-reusable-workflows/blob/main/.github/workflows/maven-cd.yml as an example

In the config you should be able to get pretty far with resolved_version if you run release drafter from git tag. Otherwise you should be able to input the version via steps input.

**Comment by slawekjaranowski** [comment]
I did something like: https://github.com/apache/maven-gh-actions-shared/blob/v4/.github/workflows/release-drafter.yml

I read version from current Maven project.
